Makefile

```diff
@@ -87,9 +88,19 @@ sync-image: sync imagebuilder
 sync-push: sync-image
 	docker push $(SYNC_IMAGE)
 
-all-image: azure-controllers-image e2e-image etcdbackup-image metricsbridge-image sync-image
+all-image: azure-controllers-image e2e-image etcdbackup-image metricsbridge-image sync-image startup-image
 
-all-push: azure-controllers-push e2e-push etcdbackup-push metrics-push sync-push
+all-push: azure-controllers-push e2e-push etcdbackup-push metrics-push sync-push startup-push
+
+startup: generate
+	go build -ldflags ${LDFLAGS} ./cmd/startup
+
+startup-image: startup
+	go get github.com/openshift/imagebuilder/cmd/imagebuilder
+	imagebuilder -f Dockerfile.startup -t $(STARTUP_IMAGE) .
+
+startup-push: startup-image
```

Comment:
I'd add startup, startup-image and startup-push to .PHONY at the end of the Makefile. Make didn't notice when source files changed for metricsbridge and sync - ended up making a new docker image with an old binary for it.

Reply:
metricsbridge needs to be in .PHONY because it does not depend on generate, which isn't in .PHONY but should be (in practice we get away with it because a file called generate does not exist).
We can get away with not listing startup, sync, etc. in .PHONY because they depend on generate.
I agree it's a bit brittle but it's good enough for this PR.